ci/build_container/Makefile

```diff
@@ -9,40 +9,40 @@ CC ?= gcc
 CXX ?= g++
 
 # Common compiler flags
-CXXFLAGS ?= "-ggdb3 -fno-omit-frame-pointer -O2"
-CFLAGS ?= "-ggdb3 -fno-omit-frame-pointer -O2"
-CPPFLAGS ?= "-DNDEBUG"
+CXXFLAGS ?= -ggdb3 -fno-omit-frame-pointer -O2
```

Comment:
I assume you tested this. It's basically impossible to visually verify this is correct or not.

Reply:
Yeah, validated